src/main/java/de/cyface/collector/handler/FormAttributes.java

```diff
@@ -1,18 +1,14 @@
 /*
  * Copyright 2018 Cyface GmbH
- * 
  * This file is part of the Cyface Data Collector.
- *
  * The Cyface Data Collector is free software: you can redistribute it and/or modify
  * it under the terms of the GNU General Public License as published by
  * the Free Software Foundation, either version 3 of the License, or
  * (at your option) any later version.
- * 
  * The Cyface Data Collector is distributed in the hope that it will be useful,
  * but WITHOUT ANY WARRANTY; without even the implied warranty of
  * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
  * GNU General Public License for more details.
- *
```

Comment:
Please leave the formatting in here. You should probably check out the most recent formatter file. I changed it so it does not remove these empty lines.

Reply:
done.